### ISSUE TITLE
Fix deploy in st2apidocs (heisenbug, since CircleCI skips deploy steps when running with SSH):

### ISSUE THREAD

**Issue by amanda11** [StackStorm/st2apidocs#13]
https://app.circleci.com/pipelines/github/StackStorm/st2apidocs/119/workflows/87289ec3-4cb3-46b2-8bd1-fe21c9f1667d/jobs/643
https://app.circleci.com/pipelines/github/StackStorm/st2apidocs/120/workflows/be123dbf-9ef3-4b20-a6ec-ac9a751577fc/jobs/647

**Comment by armab** [comment]
I see that deploy step is missing the artifacts from the generation stage:
```
find: '/root/st2apidocs/build': No such file or directory
```

which should be built during the `Generate API Documentation`. The build log looks confusing:
```
  <<< Bundling app.js
  <<< Bundling style.css
(node:204) UnhandledPromiseRejectionWarning: Error: Function yaml.safeLoad is removed in js-yaml 4. Use yaml.load instead, which is now safe by default.
    at Object.safeLoad (/usr/local/lib/node_modules/st2apidocgen/node_modules/js-yaml/index.js:10:11)
    at module.exports (/usr/local/lib/node_modules/st2apidocgen/lib/survey.js:6:21)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
(node:204) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:204) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
  >>> Done bundling app.js [f23a718c0c4f257e0bded5d0017ea925dbdd1a31dfea7e4c820ff06d9f3d4fce]
  >>> Done bundling style.css [8d66ce20197d397cfca52e3e47de12c77b70fadde97ed5ae8849be2e1f844323]
```

Worth checking if the files are really generated for the later S3 upload stage.

**Comment by cognifloyd** [comment]
Fixed with StackStorm/st2apidocsgen#11 and #14
It's working now.
